src/actions/index.js

```diff
@@ -93,9 +97,22 @@ export function refreshConfig(config) {
     }
 }
 
-/** CONTACTS */
+/** CONTACT */
```

Comment:
I love the detail here... adding the code comment sections and differentiating `contacts` from `contact` and following the camelCase to SNAKE_CASE conversion.